worlds/raft/__init__.py

```diff
@@ -115,14 +116,14 @@ def create_regions(self):
         create_regions(self.multiworld, self.player)
     
     def get_pre_fill_items(self):
-        if self.multiworld.island_frequency_locations[self.player] in [0, 1, 2, 3]:
+        if self.options.island_frequency_locations.value in [0, 1, 2, 3]:
             return [loc.item for loc in self.multiworld.get_filled_locations()]
         return []
     
     def create_item_replaceAsNecessary(self, name: str) -> Item:
         isFrequency = "Frequency" in name
-        shouldUseProgressive = ((isFrequency and self.multiworld.island_frequency_locations[self.player].value == 4)
-            or (not isFrequency and self.multiworld.progressive_items[self.player].value))
+        shouldUseProgressive = ((isFrequency and self.options.island_frequency_locations.value == 4)
+            or (not isFrequency and self.options.progressive_items.value))
```

Comment:
```suggestion
            or (not isFrequency and self.options.progressive_items))
```
Including .value isn't necessary in most spots. It's fine, just unnecessary.

Reply:
My personal recommendation is to keep the `.value` - I like the explicitness.

Reply:
I think in this case, since you are relying on conversion to bool anyway (`self.options.progressive_items.value` being an **integer**, not a boolean), I would prefer `self.options.progressive_items` since at least you're asking "what would this option be if it were a bool" instead of "what would this option's integer value representation be, if you treated it as a bool"

Reply:
Also in general I would recommend that if a class explicitly defines a conversion to something else, you should let the class do the conversion

Reply:
I personally dislike this syntax but will use it for consistency within the repo.